### ISSUE TITLE
Tooltip A11y Pass

### ISSUE THREAD

**Issue by TestAccessibilityGIT** [OfficeDev/office-ui-fabric-react#7926]
Tracking all of the issues brought up with the 2/5 accessibility pass on Icon control 
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B4e7d9198-374e-4290-9adb-25606b00a35b%7D&action=default&uid=%7B4E7D9198-374E-4290-9ADB-25606B00A35B%7D&ListItemId=1778&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble) 
  
 - [x] Bug1: [Accessibility][Programmatic access]: ARIA roles used must conform to valid values. 
 - [x] Bug2: [Accessibility][Programmatic access]: 'This is the tooltip' tooltip information is not associated with 'Tooltip without scroll' button.  
 - [x] Bug3: [Accessibility][Usable]: Narrator does not announce the tooltip information.  
 
 
 
 Component: Tooltip 
@micahgodbolt

**Comment by kenotron** [comment]
For the labelledby vs describedby issue, I'm referencing this to recommend labelledby:
https://inclusive-components.design/tooltips-toggletips/ - found that link thru a trail from here: https://www.w3.org/TR/wai-aria-practices/#tooltip

**Comment by kenotron** [comment]
Bug 3 has this note:

>Observarion:
>1. (Seems issue in Narrator in the latest built as this issue wasn’t repro in earlier versions)
>2. Issue does not repro with NVDA(2018.1.1).

Will consider this as a platform bug on Edge /Narrator's part

**Comment by v-doku** [comment]
@kenotron For Bug1 We are still getting error if we run Keros. The value which is used for aria-labelledby should match the value used for id attribute.

Please find screenshot below.
![image](https://user-images.githubusercontent.com/47735112/55715999-885a2b80-5a13-11e9-9e9c-1ec193b707f4.png)

And for Bug2 Screen reader is now reading the tooltip but not the name of button. It should read both when keyboard focus comes on it.

@micahgodbolt
